main.go

```diff
@@ -1,21 +1,45 @@
 package main
 
 import (
+	"os"
+	"time"
+
 	"github.com/RedHatInsights/entitlements-api-go/config"
 	"github.com/RedHatInsights/entitlements-api-go/controllers"
 	"github.com/RedHatInsights/entitlements-api-go/logger"
 	"github.com/RedHatInsights/entitlements-api-go/server"
 
 	"github.com/sirupsen/logrus"
+
+	"github.com/getsentry/sentry-go"
 )
 
 func main() {
+
 	// Init the logger first thing
 	logger.InitLogger()
+
+	var dsn string = os.Getenv("SENTRY_DSN")
+
+	if dsn != "" {
+		err := sentry.Init(sentry.ClientOptions{})
+		if err != nil {
+			logger.Log.WithFields(logrus.Fields{"error": err}).Error("Error loading Sentry SDK")
+		} else {
+			logger.Log.WithFields(logrus.Fields{"error": err}).Error("Sentry SDK initialization was successful!")
```

Comment:
I think we'll want this to be an `Info` log, and we shouldn't need any additional fields. Something like:
```suggestion
			logger.Log.Info("Sentry SDK initialization was successful!")
```